server/routes/liability.js

```diff
@@ -1,10 +1,55 @@
 const router = require('express').Router();
-const { Liability } = require('../db');
+const { Liability, Device, User } = require('../db').models;
 
-function check(req, res) {
-  res.send('It works!!');
+const DEFAULT_LIMIT = 30;
+
+function create(req, res, next) {
+  return Liability.create(req.body)
+    .then(liability => res.send(liability))
+    .catch(err => next(err));
+}
+
+function get(req, res, next) {
+  const where = { id: req.params.id };
+  const include = [{ model: Device }, { model: User }];
```

Comment:
Be careful with `include`. You should almost always use `as` param cause otherwise you'll end up with [uppercased property keys](http://docs.sequelizejs.com/manual/tutorial/models-usage.html#nested-eager-loading). Implement small helper that would take model name and just lowercase it.